data/campaigns/Heir_To_The_Throne/scenarios/13_The_Dwarven_Doors.cfg

```diff
@@ -236,13 +236,17 @@
         # If you have outlaws in your party, you have the chance to
         # find a special unit
         #
-        [role]
-            type=Footpad,Thug,Poacher,Outlaw,Trapper,Bandit,Fugitive,Ranger,Huntsman,Highwayman
-            role=Outlaw_Advisor
-        [/role]
         [recall]
-            role=Outlaw_Advisor
+            id=Delurin
         [/recall]
+        [role] # If Delurin was just recalled, this will select him.
```

Comment:
Because [role] checks on-map before it checks the recall list and, at this point, in prestart, we know exactly who is on the map can can match the types.